DockerfileSlave

```diff
@@ -10,8 +10,11 @@ RUN apt-get update \
     && echo deb https://apt.dockerproject.org/repo debian-jessie main >> /etc/apt/sources.list.d/docker.list \
     && apt-get update \
     && apt-get install -y docker-engine \
+    && apt-get install -y btrfs-tools \
```

Comment:
the commit 7959a23e64b00d2ace830d44583dbc895141ddd5 is misnamed, it does not only "suffix jenkins file with groovy"